data_steward/analytics/table_metrics/metrics_over_time/hpo_class.py

```diff
@@ -173,6 +180,7 @@ def print_attributes(self):
         \t Measurement Integration: {measurement_integration}\n
         \t Ingredient Integration: {ingredient_integration}\n
         \t Date/Datetime Disagreement: {date_datetime} \n
+        \t Erroneous Dates: {erroneous_dates}\n
```

Comment:
You typically don't need the extra `\n` since triple quotes print as-is. Also try to look into `f-strings`, we're starting to use them more frequently.

Reply:
Also a good idea. I'll be sure to remove the \n; didn't actually think critically about the implications of triple quotes.